backend/api/projects.py

```diff
@@ -20,13 +20,15 @@
 projects = Blueprint('api_projects', __name__)
 
 # @login_required
+
+
```

Comment:
I think we should remove the new lines at least here, the method should later be @login_required without new lines, it's not a section start as before.
It should actually be moved below the @projects.route() statement